internal/acceptance/metastore_data_access_test.go

```diff
@@ -8,7 +8,7 @@ func TestUcAccMetastoreDataAccessOnAws(t *testing.T) {
 	unityWorkspaceLevel(t, step{
 		Template: `
 		resource "databricks_metastore" "this" {
-			name          = "primary"
+			name          = "{var.RANDOM}-primary"
```

Comment:
let's add random as suffix - will be easier to visually distinguish what is test junk and what is for dev purposes

Reply:
I'll actually move this to a separate PR